### ISSUE TITLE
Add screen reader labels to the unlabeled buttons in the edit device screen of the web interface.

### ISSUE THREAD

**Issue by slannon97** [syncthing/syncthing#10873]
### What happened?

When editing a device in the web interface, there are 4 unlabeled buttons before you get to the device name box. These should have clear labels like every other button in the interface such as the save and remove buttons.

### Syncthing version

Version 2.1.3 for Windows and 2.1.2 on macOS.

### Platform & operating system

I'm using Windows 11 25H2 X86 and ARM, as well as macOS Golden Gate. The issue is present everywhere and seems to be a problem with the web interface, not a specific operating system.

### Browser version

Firefox 154.0.1 and Safari 27.0

### Relevant log output

```shell
N/A
```

**Comment by tomasz1986** [comment]
Can you be more specific which buttons you mean concretely? Maybe post a screenshot to show them?

**Edit**:

Do you mean these?

<img width="287" height="80" alt="Image" src="https://github.com/user-attachments/assets/fc6a5a3c-ae02-4e9c-a772-abb0fd844bab" />

The function is displayed in a tooltip when you hover the mouse over them (also when using the keyboard and the `TAB` key to focus them). What is the actual issue? Are you perhaps using a screen reader or is this about a touch screen?

**Comment by slannon97** [comment]
I'm totally blind and use the NVDA screen reader on Windows and VoiceOver on macOS. I'm not entirely sure what that image you sent is, but running it through AI told me there were several icons including a picture of an envelope and something about a QR code. Are these supposed to be actions you can take on the selected device?
The labels appear if I use commands to move the mouse pointer to the current control. For example, NVDA says "Copy button" or "Share by Email button" instead of just saying "button" as is the case when the edit window initially opens. I think the best solution here is to always put the labels on those controls so a screen reader user like me immediately knows what they are. I usually don't think about the mouse because it's useless in 99% of situations unless something like this pops up or a control can only be activated by moving the mouse to it and clicking rather than using standard keyboard commands like the space bar or enter after focusing it with tab or other screen reader quick navigation commands to jump to form elements like buttons. It never occurred to me to route the mouse pointer to the controls, because I only do that when I can't click on something.

**Comment by calmh** [comment]
Yeah I see the problem, these buttons are entirely unlabelled. We'll get that fixed.
